deploy/kubernetes/charts/helm-storage-backend/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+name: helm-storage-backend
+description: A Helm chart for Helm Storage Backend for Capact Local Hub
+
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.6.0
```

Comment:
**nit:** in other places we have `0.1.0` as this won't be updated on the `main` branch. IMO for consistency it will be better to change it to `0.1.0`

Reply:
This is not true: this version is updated while creating a new release. Please take a look at other charts inside `deploy/kubernetes/charts` dir.

However, this comment is true for every subchart (e.g. the ones under `capact` Helm chart).